.travis.yml

```diff
@@ -10,6 +10,8 @@ matrix:
           env: BEFORE="./bin/travis/setupnode.sh" TEST_CMD="grunt test" AFTER_SUCCESS="./bin/travis/generate_apidoc.sh"
         - php: 7.0
           env: BEFORE="composer install --prefer-dist" TEST_CMD="./vendor/phpunit/phpunit/phpunit -c phpunit.xml" AFTER_SUCCESS='echo "After success"'
+        - php: 7.0
+          env: BEFORE="./bin/travis/prepare_behat.sh" EST_CMD="php bin/behat -vv --profile=platformui" AFTER_SUCCESS='echo "After success"'
```

Comment:
Looks good, but it won't run behat because of EST (travis did not fail, but behat didn't really run).